test/Microsoft.AspNet.Mvc.FunctionalTests/ModelBindingTest.cs

```diff
@@ -2012,9 +2016,12 @@ public class ModelBindingTest
 
             // Assert
             var result = await ReadValue<PeopleModel>(response);
+
+            // Though posted content did not contain any People, it is bound as a single-element List containing
```

Comment:
1. @harshgMSFT could you file the bug about `GenericModelBinder` and some / all (??) subclasses not creating the expected empty collection in some error cases?  it really is a corner of an error case because `new[] { null }` is actually the correct model when data exists for item `0` but that data is invalid / cannot be converted to the target type. I'd rather not spend further time describing what's going on here.
2. sure I'll remove the extra spaces